src/main/java/org/opentripplanner/routing/edgetype/rentedgetype/TemporaryDropoffVehicleEdge.java

```diff
@@ -0,0 +1,11 @@
+package org.opentripplanner.routing.edgetype.rentedgetype;
+
+import org.opentripplanner.routing.edgetype.TemporaryEdge;
+import org.opentripplanner.routing.graph.Vertex;
+
+public class TemporaryDropoffVehicleEdge extends RentVehicleAnywhereEdge implements TemporaryEdge {
```

Comment:
W oryginalnym kodzie podoba mi się to, że są javadocsy mówiące do czego dana krawędź jest używana. Jest ich milion rodzajów i łatwo się pogubić

Reply:
zrobione 👍